inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_align_node.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ie_common.h>
+#include <mkldnn_node.h>
+#include <string>
+#include <memory>
+#include <vector>
+
+namespace MKLDNNPlugin {
+
+enum ROIAlignOpType {
+    Max,
+    Avg
+};
+
+class MKLDNNROIAlignNode : public MKLDNNNode {
+public:
+    MKLDNNROIAlignNode(const InferenceEngine::CNNLayerPtr& layer, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache);
+    ~MKLDNNROIAlignNode() override = default;
+
+    void getSupportedDescriptors() override;
+    void initSupportedPrimitiveDescriptors() override;
+    void createPrimitive() override;
+    void execute(mkldnn::stream strm) override;
+    bool created() const override;
+
+private:
+    int pooledH = 7;
```

Comment:
for what reason we need this default values?

Reply:
We don't need it necessary. 7 is often used for this purpose. I've used it to prevent uninitialized variable.